contrib/ansible/roles/node/tasks/main.yml

```diff
@@ -3,11 +3,10 @@
   command: getenforce
   register: selinux
   changed_when: false
-  when: ansible_selinux
 
 - name: Set selinux permissive because tokens and selinux don't work together
   selinux: state=permissive policy=targeted
-  when: ansible_selinux and 'Enforcing' in selinux.stdout
+  when: "'Enforcing' in selinux.stdout"
```

Comment:
No, you can't do this. We shouldn't be calling getenforce as it might not be installed on non-selinux systems. I think we want to get rid of the 'getenforce' command entirely and use the ansible_selinux facts entirely in the when portion of `Set selinux permissive because tokens and selinux don't work together`